### PR TITLE
Update Flight Hub to add IoT, ISO & SDK builds for the RS2 section

### PR DESCRIPTION
**Pull request by ItzLevvie**
from https://github.com/MicrosoftDocs/windows-insider/pull/224

This PR adds:
* Added all IoT, ISO & SDK builds that have went off with **RS2** flighting.

**_EDIT: It seems like this change may have been reverted previously due to a conflict_**:
My RS2 change from PR was added in https://github.com/MicrosoftDocs/windows-insider/commit/103a90ff959426900456ddbd6a2077754688d130, was good until https://github.com/MicrosoftDocs/windows-insider/commit/c7b57d55eb0428ab528551a98cedf0a7cc3ca5a1, but got reverted in https://github.com/MicrosoftDocs/windows-insider/commit/27ec156dd1c7c9cbd9a160efe8be4d7d465dd00a after my RS1 PR got merged.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: wip/flight-hub/index.md [hunk 1]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|15061|[3/17/2017](https://blogs.windows.com/windowsexperience/2017/03/17/announcing-windows-10-insider-preview-build-15061-pc/)||||||
+|15060|[3/16/2017](https://blogs.windows.com/windowsexperience/2017/03/16/announcing-windows-10-insider-preview-build-15060-pc/)||||||
+|15058|[3/14/2017](https://blogs.windows.com/windowsexperience/2017/03/14/announcing-windows-10-insider-preview-build-15058-pc/)|[3/17/2017](https://blogs.windows.com/windowsexperience/2017/03/14/announcing-windows-10-insider-preview-build-15058-pc/)||[3/16/2017](https://social.msdn.microsoft.com/Forums/en-US/696ed681-b9cc-4d25-bc6c-86bdb47bcf5c/new-flight-15058-released-for-insiders)|[3/17/2017](https://www.microsoft.com/software-download/windowsinsiderpreviewadvanced)||
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|15058|[3/14/2017](https://blogs.windows.com/windowsexperience/2017/03/14/announcing-windows-10-insider-preview-build-15058-pc/)|[3/17/2017](https://blogs.windows.com/windowsexperience/2017/03/14/announcing-windows-10-insider-preview-build-15058-pc/)||[3/16/2017](https://social.msdn.microsoft.com/Forums/696ed681-b9cc-4d25-bc6c-86bdb47bcf5c/new-flight-15058-released-for-insiders)|[3/17/2017](https://www.microsoft.com/software-download/windowsinsiderpreviewadvanced)||
```

File: wip/flight-hub/index.md [hunk 2]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|15055|[3/10/2017](https://blogs.windows.com/windowsexperience/2017/03/10/announcing-windows-10-insider-preview-build-15055-pc-mobile/)||||||
+|15052||||||[3/14/2017](https://blogs.windows.com/windowsdeveloper/2017/03/14/windows-10-sdk-preview-build-15052-released/)|
+|15051|[3/8/2017](https://blogs.windows.com/windowsexperience/2017/03/08/announcing-windows-10-insider-preview-build-15051-mobile/)|[3/10/2017](https://blogs.windows.com/windowsexperience/2017/03/08/announcing-windows-10-insider-preview-build-15051-mobile/)||[3/9/2017](https://social.msdn.microsoft.com/Forums/en-US/2e96b5f7-84ed-4f54-9bfd-60c2b050942a/new-flight-15051-released-for-insiders)|||
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|15051|[3/8/2017](https://blogs.windows.com/windowsexperience/2017/03/08/announcing-windows-10-insider-preview-build-15051-mobile/)|[3/10/2017](https://blogs.windows.com/windowsexperience/2017/03/08/announcing-windows-10-insider-preview-build-15051-mobile/)||[3/9/2017](https://social.msdn.microsoft.com/Forums/2e96b5f7-84ed-4f54-9bfd-60c2b050942a/new-flight-15051-released-for-insiders)|||
```

File: wip/flight-hub/index.md [hunk 3]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|15048|[3/3/2017](https://blogs.windows.com/windowsexperience/2017/03/03/announcing-windows-10-insider-preview-build-15048-for-pc-and-build-15047-mobile/)|[3/8/2017](https://blogs.windows.com/windowsexperience/2017/03/03/announcing-windows-10-insider-preview-build-15048-for-pc-and-build-15047-mobile/)|||[3/14/2017](https://www.microsoft.com/software-download/windowsinsiderpreviewadvanced)||
+|15047|[3/3/2017](https://blogs.windows.com/windowsexperience/2017/03/03/announcing-windows-10-insider-preview-build-15048-for-pc-and-build-15047-mobile/)||||||
+|15046|[2/28/2017](https://blogs.windows.com/windowsexperience/2017/02/28/announcing-windows-10-insider-preview-build-15046-pc/)||||||
+|15043|[2/24/2017](https://blogs.windows.com/windowsexperience/2017/02/24/announcing-windows-10-insider-preview-build-15042-pc-build-15043-mobile/)|[3/2/2017](https://blogs.windows.com/windowsexperience/2017/02/24/announcing-windows-10-insider-preview-build-15042-pc-build-15043-mobile/)||[2/25/2017](https://social.msdn.microsoft.com/Forums/en-US/1772b05f-dd3e-4182-86ff-9e374f0aaf32/new-flight-15043-released-for-insiders)|||
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|15043|[2/24/2017](https://blogs.windows.com/windowsexperience/2017/02/24/announcing-windows-10-insider-preview-build-15042-pc-build-15043-mobile/)|[3/2/2017](https://blogs.windows.com/windowsexperience/2017/02/24/announcing-windows-10-insider-preview-build-15042-pc-build-15043-mobile/)||[2/25/2017](https://social.msdn.microsoft.com/Forums/1772b05f-dd3e-4182-86ff-9e374f0aaf32/new-flight-15043-released-for-insiders)|||
```

File: wip/flight-hub/index.md [hunk 4]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|15025|[2/2/2017](https://blogs.windows.com/windowsexperience/2017/02/02/announcing-windows-10-insider-preview-build-15025-mobile/)<br>[2/1/2017](https://blogs.windows.com/windowsexperience/2017/02/01/announcing-windows-10-insider-preview-build-15025-pc/)||||[2/5/2017](https://www.microsoft.com/software-download/windowsinsiderpreviewadvanced)||
+|15021||||||[2/3/2017](https://blogs.windows.com/windowsdeveloper/2017/02/03/windows-10-sdk-preview-build-15021-released/)|
+|15019|[1/27/2017](https://blogs.windows.com/windowsexperience/2017/01/27/announcing-windows-10-insider-preview-build-15019-pc/)||||||
+|15014|[1/19/2017](https://blogs.windows.com/windowsexperience/2017/01/19/announcing-windows-10-insider-preview-build-15014-for-pc-and-mobile-hello-windows-insiders-today-we-are-excited-to-be-releasing-windows-10-insider-preview-build-15014-for-pc-and-mobile/)||||||
+|15007|[1/12/2017](https://blogs.windows.com/windowsexperience/2017/01/12/announcing-windows-10-insider-preview-build-15007-pc-mobile/)|||[1/12/2017](https://social.msdn.microsoft.com/Forums/en-US/7c3dd88a-b3f0-46c4-bbcd-f3ec21891380/new-flight-15007-released-for-insiders)|||
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|15007|[1/12/2017](https://blogs.windows.com/windowsexperience/2017/01/12/announcing-windows-10-insider-preview-build-15007-pc-mobile/)|||[1/12/2017](https://social.msdn.microsoft.com/Forums/7c3dd88a-b3f0-46c4-bbcd-f3ec21891380/new-flight-15007-released-for-insiders)|||
```

File: wip/flight-hub/index.md [hunk 5]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|15003||||||[1/13/2017](https://blogs.windows.com/windowsdeveloper/2017/01/13/windows-10-sdk-preview-build-15003-released/)|
+|15002|[1/9/2017](https://blogs.windows.com/windowsexperience/2017/01/09/announcing-windows-10-insider-preview-build-15002-pc/)|||[1/9/2017](https://social.msdn.microsoft.com/Forums/en-US/b4b2f11d-3318-4efa-8e8b-778ece599046/new-flight-15002-released-for-insiders)|[1/13/2017](https://www.microsoft.com/software-download/windowsinsiderpreviewadvanced)||
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|15002|[1/9/2017](https://blogs.windows.com/windowsexperience/2017/01/09/announcing-windows-10-insider-preview-build-15002-pc/)|||[1/9/2017](https://social.msdn.microsoft.com/Forums/b4b2f11d-3318-4efa-8e8b-778ece599046/new-flight-15002-released-for-insiders)|[1/13/2017](https://www.microsoft.com/software-download/windowsinsiderpreviewadvanced)||
```

File: wip/flight-hub/index.md [hunk 6]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|14986|[12/7/2016](https://blogs.windows.com/windowsexperience/2016/12/07/announcing-windows-10-insider-preview-build-14986-pc/)|[12/14/2016](https://blogs.windows.com/windowsexperience/2016/12/07/announcing-windows-10-insider-preview-build-14986-pc/)||[12/10/2016](https://social.msdn.microsoft.com/Forums/en-US/be5fc892-b4d7-4f44-a3c2-c0a1c302dc0c/new-flight-14986-released-for-insiders)|||
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|14986|[12/7/2016](https://blogs.windows.com/windowsexperience/2016/12/07/announcing-windows-10-insider-preview-build-14986-pc/)|[12/14/2016](https://blogs.windows.com/windowsexperience/2016/12/07/announcing-windows-10-insider-preview-build-14986-pc/)||[12/10/2016](https://social.msdn.microsoft.com/Forums/be5fc892-b4d7-4f44-a3c2-c0a1c302dc0c/new-flight-14986-released-for-insiders)|||
```

File: wip/flight-hub/index.md [hunk 7]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|14977|[12/1/2016](https://blogs.windows.com/windowsexperience/2016/12/01/announcing-windows-10-insider-preview-build-14977-mobile/)||||||
+|14971|[11/17/2016](https://blogs.windows.com/windowsexperience/2016/11/17/announcing-windows-10-insider-preview-build-14971-for-pc/)|||[11/17/2016](https://social.msdn.microsoft.com/Forums/en-US/1d5244ac-bf5c-486c-a203-b0406bc858f6/new-flight-14971-released-for-insiders)|||
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|14971|[11/17/2016](https://blogs.windows.com/windowsexperience/2016/11/17/announcing-windows-10-insider-preview-build-14971-for-pc/)|||[11/17/2016](https://social.msdn.microsoft.com/Forums/1d5244ac-bf5c-486c-a203-b0406bc858f6/new-flight-14971-released-for-insiders)|||
```

File: wip/flight-hub/index.md [hunk 8]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|14965|[11/9/2016](https://blogs.windows.com/windowsexperience/2016/11/09/announcing-windows-10-insider-preview-build-14965-for-mobile-and-pc/)|[11/17/2016](https://blogs.windows.com/windowsexperience/2016/11/09/announcing-windows-10-insider-preview-build-14965-for-mobile-and-pc/)<br>[11/16/2016](https://blogs.windows.com/windowsexperience/2016/11/09/announcing-windows-10-insider-preview-build-14965-for-mobile-and-pc/)||[11/10/2016](https://social.msdn.microsoft.com/Forums/en-US/76bdff34-8816-4b48-aa36-c91b1f2c2447/new-flight-14965-released-for-insiders)||[11/21/2016](https://blogs.windows.com/windowsdeveloper/2016/11/21/windows-10-sdk-preview-build-14965-released/)|
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|14965|[11/9/2016](https://blogs.windows.com/windowsexperience/2016/11/09/announcing-windows-10-insider-preview-build-14965-for-mobile-and-pc/)|[11/17/2016](https://blogs.windows.com/windowsexperience/2016/11/09/announcing-windows-10-insider-preview-build-14965-for-mobile-and-pc/)<br>[11/16/2016](https://blogs.windows.com/windowsexperience/2016/11/09/announcing-windows-10-insider-preview-build-14965-for-mobile-and-pc/)||[11/10/2016](https://social.msdn.microsoft.com/Forums/76bdff34-8816-4b48-aa36-c91b1f2c2447/new-flight-14965-released-for-insiders)||[11/21/2016](https://blogs.windows.com/windowsdeveloper/2016/11/21/windows-10-sdk-preview-build-14965-released/)|
```

File: wip/flight-hub/index.md [hunk 9]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|14959|[11/3/2016](https://blogs.windows.com/windowsexperience/2016/11/03/announcing-windows-10-insider-preview-build-14959-for-mobile-and-pc/)|||[11/3/2016](https://social.msdn.microsoft.com/Forums/en-US/a6c9acaf-6b4c-445f-aef2-041ea743061e/new-flight-14959-released-for-insiders)|||
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|14959|[11/3/2016](https://blogs.windows.com/windowsexperience/2016/11/03/announcing-windows-10-insider-preview-build-14959-for-mobile-and-pc/)|||[11/3/2016](https://social.msdn.microsoft.com/Forums/a6c9acaf-6b4c-445f-aef2-041ea743061e/new-flight-14959-released-for-insiders)|||
```

File: wip/flight-hub/index.md [hunk 10]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|14955|[10/25/2016](https://blogs.windows.com/windowsexperience/2016/10/25/announcing-windows-10-insider-preview-build-14955-for-mobile-and-pc/)|||[11/2/2016](https://social.msdn.microsoft.com/Forums/en-US/3c6bb41c-6f1d-4e11-ba3a-fac208721bc7/new-flight-14955-released)|||
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|14955|[10/25/2016](https://blogs.windows.com/windowsexperience/2016/10/25/announcing-windows-10-insider-preview-build-14955-for-mobile-and-pc/)|||[11/2/2016](https://social.msdn.microsoft.com/Forums/3c6bb41c-6f1d-4e11-ba3a-fac208721bc7/new-flight-14955-released)|||
```

File: wip/flight-hub/index.md [hunk 11]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|14951|[10/19/2016](https://blogs.windows.com/windowsexperience/2016/10/19/announcing-windows-10-insider-preview-build-14951-for-mobile-and-pc/)|||[10/21/2016](https://social.msdn.microsoft.com/Forums/en-US/f2170a43-dd3c-45bd-84fc-471508ba9364/new-flight-14951-released)|||
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|14951|[10/19/2016](https://blogs.windows.com/windowsexperience/2016/10/19/announcing-windows-10-insider-preview-build-14951-for-mobile-and-pc/)|||[10/21/2016](https://social.msdn.microsoft.com/Forums/f2170a43-dd3c-45bd-84fc-471508ba9364/new-flight-14951-released)|||
```

File: wip/flight-hub/index.md [hunk 12]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|14946|[10/13/2016](https://blogs.windows.com/windowsexperience/2016/10/13/announcing-windows-10-insider-preview-build-14946-for-pc-and-mobile/)||||||
+|14942|[10/7/2016](https://blogs.windows.com/windowsexperience/2016/10/07/announcing-windows-10-insider-preview-build-14942-for-pc/)||||||
+|14936|[9/28/2016](https://blogs.windows.com/windowsexperience/2016/09/28/announcing-windows-10-insider-preview-build-14936-for-pc/)|||[9/29/2016](https://social.msdn.microsoft.com/Forums/en-US/9c92fe44-166c-4ae9-9c66-09553315d97a/new-flight-14936-released)|||
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|14936|[9/28/2016](https://blogs.windows.com/windowsexperience/2016/09/28/announcing-windows-10-insider-preview-build-14936-for-pc/)|||[9/29/2016](https://social.msdn.microsoft.com/Forums/9c92fe44-166c-4ae9-9c66-09553315d97a/new-flight-14936-released)|||
```

File: wip/flight-hub/index.md [hunk 13]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|14931|[9/21/2016](https://blogs.windows.com/windowsexperience/2016/09/21/announcing-windows-10-insider-preview-build-14931-for-pc/)|[10/5/2016](https://blogs.windows.com/windowsexperience/2016/09/21/announcing-windows-10-insider-preview-build-14931-for-pc/)||[9/22/2016](https://social.msdn.microsoft.com/Forums/en-US/163e8227-6e5a-43a5-9f69-499304587a4b/new-flight-14931-released)|[10/5/2016](https://www.microsoft.com/software-download/windowsinsiderpreviewadvanced)||
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|14931|[9/21/2016](https://blogs.windows.com/windowsexperience/2016/09/21/announcing-windows-10-insider-preview-build-14931-for-pc/)|[10/5/2016](https://blogs.windows.com/windowsexperience/2016/09/21/announcing-windows-10-insider-preview-build-14931-for-pc/)||[9/22/2016](https://social.msdn.microsoft.com/Forums/163e8227-6e5a-43a5-9f69-499304587a4b/new-flight-14931-released)|[10/5/2016](https://www.microsoft.com/software-download/windowsinsiderpreviewadvanced)||
```

File: wip/flight-hub/index.md [hunk 14]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|14926|[9/14/2016](https://blogs.windows.com/windowsexperience/2016/09/14/announcing-windows-10-insider-preview-build-14926-for-pc-and-mobile/)|||[9/19/2016](https://social.msdn.microsoft.com/Forums/en-US/9f7c7042-5b49-4563-8385-89bd21b0feb7/new-flight-14926-released)|||
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|14926|[9/14/2016](https://blogs.windows.com/windowsexperience/2016/09/14/announcing-windows-10-insider-preview-build-14926-for-pc-and-mobile/)|||[9/19/2016](https://social.msdn.microsoft.com/Forums/9f7c7042-5b49-4563-8385-89bd21b0feb7/new-flight-14926-released)|||
```

File: wip/flight-hub/index.md [hunk 15]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
+|14915|[8/31/2016](https://blogs.windows.com/windowsexperience/2016/08/31/announcing-windows-10-insider-preview-build-14915-for-pc-and-mobile/)|||[9/6/2016](https://social.msdn.microsoft.com/Forums/en-US/4ddb5e1d-fbf1-46e0-9b47-5dfd24202826/new-flight-14915-released)|||
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|14915|[8/31/2016](https://blogs.windows.com/windowsexperience/2016/08/31/announcing-windows-10-insider-preview-build-14915-for-pc-and-mobile/)|||[9/6/2016](https://social.msdn.microsoft.com/Forums/4ddb5e1d-fbf1-46e0-9b47-5dfd24202826/new-flight-14915-released)|||
```

File: wip/flight-hub/index.md [hunk 16]
```diff
@@ -241,40 +241,43 @@ The [Windows 10 Fall Creators Update](https://blogs.windows.com/windowsexperienc
...
-|14905|[8/17/2016](https://blogs.windows.com/windowsexperience/2016/08/17/announcing-windows-10-insider-preview-build-14905-for-pc-mobile/)|||
-|14901|[8/11/2016](https://blogs.windows.com/windowsexperience/2016/08/11/announcing-windows-10-insider-preview-build-14901-for-pc/)|||
+| Build | Fast | Slow | Release Preview | IoT | ISO | SDK |
+|-------|------|------|-----------------|-----|-----|-----|
+|15063|[**3/20/2017**](https://blogs.windows.com/windowsexperience/2017/03/20/announcing-windows-10-insider-preview-build-15063-pc-mobile/)|[**3/23/2017**](https://blogs.windows.com/windowsexperience/2017/03/20/announcing-windows-10-insider-preview-build-15063-pc-mobile/)|[**3/30/2017**](https://blogs.windows.com/windowsexperience/2017/03/20/announcing-windows-10-insider-preview-build-15063-pc-mobile/)|[**3/23/2017**](https://social.msdn.microsoft.com/Forums/en-US/55da1cc7-a852-41d3-a04f-ed9fd3ba5666/new-flight-15063-released-for-insiders)|[**3/30/2017**](https://www.microsoft.com/software-download/windowsinsiderpreviewadvanced)|[**4/5/2017**](https://blogs.windows.com/windowsdeveloper/2017/04/05/windows-10-creators-update-creators-update-sdk-released/)|
```
Comment (JohanFreelancer9) [inline wip/flight-hub/index.md]:
```suggestion
|15063|[**3/20/2017**](https://blogs.windows.com/windowsexperience/2017/03/20/announcing-windows-10-insider-preview-build-15063-pc-mobile/)|[**3/23/2017**](https://blogs.windows.com/windowsexperience/2017/03/20/announcing-windows-10-insider-preview-build-15063-pc-mobile/)|[**3/30/2017**](https://blogs.windows.com/windowsexperience/2017/03/20/announcing-windows-10-insider-preview-build-15063-pc-mobile/)|[**3/23/2017**](https://social.msdn.microsoft.com/Forums/55da1cc7-a852-41d3-a04f-ed9fd3ba5666/new-flight-15063-released-for-insiders)|[**3/30/2017**](https://www.microsoft.com/software-download/windowsinsiderpreviewadvanced)|[**4/5/2017**](https://blogs.windows.com/windowsdeveloper/2017/04/05/windows-10-creators-update-creators-update-sdk-released/)|
```

**Comment by ItzLevvie** [comment]
cc @JohanFreelancer9 the links don't seem to work when the locale is removed.

**Comment by mypil** [comment]
@lizap - PR is ready for final review. 

 Thank you.

cc: @kenwith

**Comment by ItzLevvie** [comment]
cc @lizap

**Comment by mypil** [comment]
adding @bleblancMSFT to the assignee as suggested by lizap to help review and merge this PR.

Thank you.

cc: @kenwith

**Comment by ItzLevvie** [comment]
cc @bleblancMSFT

**Comment by bleblancMSFT** [comment]
@ItzLevvie - can you hold off going back and adding updates for prior releases like RS4-TH1 etc.? We're doing some updates and adjustments we want to ship and some of the items you're adding doesn't match up with our records. Also - we do not want to add Mobile builds to Flight Hub.

**Comment by ItzLevvie** [comment]
@bleblancMSFT Alright.

**Comment by bleblancMSFT** [comment]
@ItzLevvie I added in the 19002 SDK release and latest 19H2 releases. Let me know if it looks right!

**Comment by ItzLevvie** [comment]
@bleblancMSFT Just a spelling mistake needs to be fixed. I have opened a PR over at https://github.com/MicrosoftDocs/windows-insider/pull/242. Other than that, it looks good :)

**Comment by bleblancMSFT** [comment]
@ItzLevvie just approved and did the merge! THANK YOU!

**Pull request merged**